jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/Dispatcher.java

```diff
@@ -620,19 +619,30 @@ public DispatcherType getDispatcherType()
         @Override
         public String getPathInfo()
         {
+            if (_servletPathMapping == null)
+                return super.getPathInfo();
```

Comment:
what is the situation when we have an error dispatch with `_servletPathMapping == null`?  I don't think that is possible?

Reply:
I thought only if there was no Default servlet on `/` url-pattern.
Turns out that's not a case either.

The `ErrorHandler.handle()` won't dispatch if the `RequestDispatcher` cannot find a page to dispatch to.
Meaning that even if the default servlet is unset, and something like `/doesnotexist` is requested, no `Dispatcher$ErrorRequest` is ever created.
The `ErrorHandler.handle()` will just return a generic error response if there is no match/mapping.

I changed all of these to `Objects.requireNonNull(_servletPathMapping)`